### PR TITLE
Solana, solana tx hash, signing, broadcasting

### PR DESCRIPTION
**Pull request by enriquesouza**

### CODE REVIEW

File: frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts [hunk 1]
```diff
@@ -263,30 +327,25 @@ export class BlockchainServiceSolana
...
     const output = TW.Solana.Proto.SigningOutput.decode(compiled);
-
     if (output.errorMessage !== '') {
       throw new Error(output.errorMessage);
     }
 
     const result = new SignedTransactionResult(
       output.encoded,
-      this.getHashFromRawTransaction(output.encoded)
+      output.encoded // TODO: Change this to the actual transaction hash
```
Comment (johnnyluo) [inline frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts:344]: this one is still `TODO`

**Review by johnnyluo — APPROVED** [review]
LGTM

File: frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts (hunk 1 above)
Comment (enriquesouza) [reply frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts:344]: Yes, I didn't find a way to do it. It does not match with the other device.

Comment (enriquesouza) [reply frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts:344]: This logic here does not match with the other device:

```
static func getHashFromRawTransaction(tx: String) -> String {
        let sig =  Data(tx.prefix(64).utf8)
        return sig.base64EncodedString()
    }
    ```
    
    Therefore once we broadcast the RPC returns the correct one, no this one is not needed actually.

**Pull request merged**